oecologia-australis.csl

```diff
@@ -61,21 +61,17 @@
     <choose>
       <if type="article-journal chapter paper-conference entry-dictionary entry-encyclopedia" match="none">
         <group delimiter=", ">
-          <names variable="container-author" delimiter=", ">
-            <name and="symbol" initialize-with=". " delimiter=", "/>
-            <label form="short" prefix=", " text-case="title"/>
-          </names>
-          <names variable="editor translator" delimiter=", ">
-            <name and="symbol" initialize-with=". " delimiter=", "/>
-            <label form="short" prefix=", " text-case="title"/>
+          <names variable="editor translator" delimiter=", " suffix=" ">
+            <name delimiter=" " suffix="," and="symbol" initialize-with=". "/>
+            <label form="short" text-case="title" prefix=" "/>
           </names>
         </group>
       </if>
     </choose>
   </macro>
   <macro name="author">
     <names variable="author">
-      <name and="symbol" delimiter-precedes-last="always" initialize-with=". " name-as-sort-order="all"/>
+      <name and="symbol" delimiter-precedes-last="always" et-al-min="1" et-al-use-first="0" initialize-with=". " name-as-sort-order="all"/>
```

Comment:
@gudryan, these values for the "et-al-..." attributes make no sense. Is "et al." even used in the bibliography, where the "author" macro is used?

See http://docs.citationstyles.org/en/stable/specification.html#name for a description of the attributes.